etc/profile-m-z/xlinks.profile

```diff
@@ -1,21 +1,18 @@
 # Firejail profile for xlinks
 # Description: Text WWW browser (X11)
 # This file is overwritten after every install/update
+quiet
 # Persistent local customizations
 include xlinks.local
 # Persistent global definitions
-# added by included profile
-#include globals.local
+include globals.local
 
-noblacklist /tmp/.X11-unix
-noblacklist ${HOME}/.links
+noblacklist ${HOME}/.xlinks
 
-include whitelist-common.inc
+mkdir ${HOME}/.xlinks
+whitelist ${HOME}/.xlinks
 
-# if you want to use user-configured programs add 'private-bin PROGRAM1,PROGRAM2'
-# to your xlinks.local or append 'PROGRAM1,PROGRAM2' to this private-bin line
 private-bin xlinks
-private-etc fonts
```

Comment:
This makes the font ugly, doesn't it?

Reply:
i can't locate xlinks

Reply:
`locate(1)`?

Reply:
links can be compiled with an optional graphical interface( -g option). I think xlinks is just a recompile of some distro, with the graphical interface enabled. links2 seams to be the same thing in debian/ubuntu (the package links has no -g option). Probably in others, they just compiled it in links it self...

Reply:
xlinks is a wrapper script for "links -g"
reverting